sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.bulkInsert.e2e-spec.ts

```diff
@@ -221,35 +221,38 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-bulkInsert", () =>
         .attach("file", singleOfferingWithValidationErrorsFilePath)
         .auth(institutionUserToken, BEARER_AUTH_TYPE)
         .expect(HttpStatus.UNPROCESSABLE_ENTITY)
-        .expect({
-          message: "An offering has invalid data.",
-          errorType: OFFERING_VALIDATION_CRITICAL_ERROR,
-          objectInfo: [
-            {
-              recordIndex: 0,
-              locationCode: csvLocationCodeYESK,
-              sabcProgramCode: csvProgramSABCCodeSBC1,
-              startDate: "2023-09-05",
-              endDate: "2023-10-14",
-              offeringStatus: OfferingStatus.CreationPending,
-              errors: [],
-              infos: [],
-              warnings: [
-                {
-                  typeCode:
-                    OfferingValidationWarnings.InvalidStudyDatesPeriodLength,
-                  message:
-                    "End date, the number of day(s) between Sep 05 2023 and Oct 14 2023 must be at least 84.",
-                },
-                {
-                  typeCode:
-                    OfferingValidationWarnings.ProgramOfferingDeliveryMismatch,
-                  message:
-                    "Delivery type has an offering delivery that is not allowed by its program.",
-                },
-              ],
-            },
-          ],
+        .expect((response) => {
+          console.log("response.body: ", response.body);
+          expect(response.body).toEqual({
+            message: "An offering has invalid data.",
+            errorType: OFFERING_VALIDATION_CRITICAL_ERROR,
+            objectInfo: [
+              {
+                recordIndex: 0,
+                locationCode: csvLocationCodeYESK,
+                sabcProgramCode: csvProgramSABCCodeSBC1,
+                startDate: "2023-09-05",
+                endDate: "2023-10-14",
+                offeringStatus: OfferingStatus.CreationPending,
+                errors: [],
+                infos: [],
+                warnings: expect.arrayContaining([
+                  expect.objectContaining({
+                    typeCode:
+                      OfferingValidationWarnings.InvalidFundedStudyPeriodLength,
+                    message:
+                      "The funded study amount of days is ineligible for StudentAid BC funding. Your offering must be at least 12 weeks of study or longer to be eligible.",
```

Comment:
I made a comment previously regarding the message having `funded study amount of days`